src/backend/db/products.js

```diff
@@ -25,6 +26,7 @@ export const products = [
     description: "Printed Black T-Shirt",
     rating: "5",
```

Comment:
Add an ID key as well.

Reply:
yeh forgot that thanks for pointing it out Sakshi